### ISSUE TITLE
If several fields with same name have default filter on, they show multiple times in the sidepanel form

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazi#1632]
If several templates have a property with the same name and set as default filters, they show multiples times in the sidepanel.  This could be solved by showing it just once, but this needs to be type-aware.  Becasue if there are fileds with same name that have different types (select and multiselect, for instance) then we need to figure the correct way of doing this.

Finally, there is a bug that, even if the document type is filtered and selected, still there are multiple properties of the same type showing, which is plain wrong.  The SASCL database is a good example for this.

**Comment by txau** [comment]
@RafaPolit ideally default filters behave just as regular filters. If templates have a property with the same name and type, the filter is just rendered once for all templates.

**Comment by RafaPolit** [comment]
Yes, the filter reducing is working as expected.  Please read #1633 for description of the underlying problem.

The duplicate key error has been fixed in PR.

**Comment by txau** [comment]
@rafapolit regarding the common filter for select and multi-select. I guess that should not affect and still be the same filter. I don't see any drawbacks in that sense. The criteria should be by name and thesaurus.
